provisioning/e2e/_service_create_delete.js

```diff
@@ -92,4 +94,29 @@ testDescription.forEach(function(testConfiguration) {
       });
     });
   });
-});
\ No newline at end of file
+});
+
+testSpecification.forEach(function(testConfiguration) {
+  describe('#Update', function() {
+    this.timeout(10000);
+    var enrollmentToDelete = {};
+    after(function(done) {
+      testConfiguration.deleteFunction(enrollmentToDelete, function(err) {
+        assert.isNull(err,'Non null response from the delete AFTER create.');
+        done();
+      });
+    });
+    it(testConfiguration.testDescription, function(callback) {
+      enrollmentToDelete = {};
+      testConfiguration.createFunction(testConfiguration.enrollmentObject, function(err, returnedEnrollment) {
+        assert.isNull(err,'Should be no error from the create');
+        returnedEnrollment.provisioningState = 'disabled';
+        testConfiguration.updateFunction(returnedEnrollment, function(err, updatedEnrollment) {
```

Comment:
>updateFunction [](start = 26, length = 14)

missing verification that the updatedEnrollment is now disabled.

Reply:
Fixed.

---
In reply to: [154166074](https://github.com/Azure/azure-iot-sdk-node/pull/165#discussion_r154166074) [](ancestors = 154166074)